cmd/signozschemamigrator/schema_migrator/traces_migrations.go

```diff
@@ -1,3 +1,360 @@
+					{Name: "attribute_string_messaging$$operation", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "msgOperation"},
+					{Name: "attribute_string_db$$system", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "dbSystem"},
+					{Name: "attribute_string_rpc$$system", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "rpcSystem"},
+					{Name: "attribute_string_rpc$$service", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "rpcService"},
+					{Name: "attribute_string_rpc$$method", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "rpcMethod"},
+					{Name: "attribute_string_peer$$service", Type: LowCardinalityColumnType{ColumnTypeString}, Alias: "peerService"},
+				},
+				Engine: Distributed{
+					Database:    "signoz_traces",
+					Table:       "signoz_index_v3",
+					ShardingKey: "cityHash64(traceID)",
+				},
+			},
+			CreateTableOperation{
+				Database: "signoz_traces",
+				Table:    "traces_v3_resource",
+				Columns: []Column{
+					{Name: "labels", Type: ColumnTypeString, Codec: "ZSTD(5)"},
+					{Name: "fingerprint", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "seen_at_ts_bucket_start", Type: ColumnTypeInt64, Codec: "Delta(8), ZSTD(1)"},
+				},
+				Indexes: []Index{
+					{Name: "idx_labels", Expression: "lower(labels)", Type: "ngrambf_v1(4, 1024, 3, 0)", Granularity: 1},
+					{Name: "idx_labels_v1", Expression: "labels", Type: "ngrambf_v1(4, 1024, 3, 0)", Granularity: 1},
+				},
+				Engine: ReplacingMergeTree{
+					MergeTree: MergeTree{
+						PartitionBy: "toDate(seen_at_ts_bucket_start / 1000)",
+						OrderBy:     "(labels, fingerprint, seen_at_ts_bucket_start)",
+						TTL:         "toDateTime(seen_at_ts_bucket_start) + INTERVAL 1296000 SECOND + INTERVAL 1800 SECOND DELETE",
+						Settings: TableSettings{
+							{Name: "ttl_only_drop_parts", Value: "1"},
+							{Name: "index_granularity", Value: "8192"},
+						},
+					},
+				},
+			},
+			CreateTableOperation{
+				Database: "signoz_traces",
+				Table:    "distributed_traces_v3_resource",
+				Columns: []Column{
+					{Name: "labels", Type: ColumnTypeString, Codec: "ZSTD(5)"},
+					{Name: "fingerprint", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "seen_at_ts_bucket_start", Type: ColumnTypeInt64, Codec: "Delta(8), ZSTD(1)"},
+				},
+				Engine: Distributed{
+					Database:    "signoz_traces",
+					Table:       "traces_v3_resource",
+					ShardingKey: "cityHash64(labels, fingerprint)",
+				},
+			},
+			CreateTableOperation{
+				Database: "signoz_traces",
+				Table:    "trace_summary",
+				Columns: []Column{
+					{Name: "traceID", Type: ColumnTypeString, Codec: "ZSTD(1)"},
+					{Name: "first_reported",
+						Type: SimpleAggregateFunction{
+							FunctionName: "min",
+							Arguments:    []ColumnType{DateTime64ColumnType{Precision: 9}},
+						},
+						Codec: "ZSTD(1)"},
+					{Name: "last_reported",
```

Comment:
Is it the reported time of the collector writing to the table, or is the max timestamp of trace?

Reply:
it's max timestamp of the trace

Reply:
maybe rename it because it reads like a trace reporting time than max timestamp of the trace

Reply:
stand and end will be better I guess.